ticketbookingapp/src/main/java/pl/touk/recruitmenttask/ticketbookingapp/service/properties/PropertiesConfig.java

```diff
@@ -0,0 +1,11 @@
+package pl.touk.recruitmenttask.ticketbookingapp.service.properties;
+
+public class PropertiesConfig {
+    public final static double adultPrice = 25;
```

Comment:
jak ktoś postanowi zmienić cenę to musimy przekompilować kod?

Reply:
Niestety tak, ze względu, że wartości te są compile-time constants.